saleor/dashboard/staff/forms.py

```diff
@@ -17,7 +17,7 @@ class StaffForm(forms.ModelForm):
     class Meta:
         model = User
         fields = ['first_name', 'last_name', 'email',
-                  'user_permissions', 'is_active']
+                  'user_permissions', 'is_active', 'is_staff', 'is_superuser']
```

Comment:
"Superuser" functionality is hidden in UI on purpose and we don't want to reveal that. We think that superusers are convenient for development but normally, regular staff accounts and permission systems should be used.

Reply:
Also, it's a rare case to turn your customers into staff - that's why this is hidden as well. However, if you really wanted to do that, go to the "staff create" view and try to add a staff user with an email of existing customer - this action will succeed and the user will receive "Promote to staff" email.

Reply:
> Also, it's a rare case to turn your customers into staff - that's why this is hidden as well. However, if you really wanted to do that, go to the "staff create" view and try to add a staff user with an email of existing customer - this action will succeed and the user will receive "Promote to staff" email.

OK, sounds good, I will remove is_staff in customer form

Reply:
> "Superuser" functionality is hidden in UI on purpose and we don't want to reveal that. We think that superusers are convenient for development but normally, regular staff accounts and permission systems should be used.

Maybe at least we should reveal which user is superuser? I found it's confusing in some API that I have permission but I shouldn't have. At last I found I'm a superuser. :D

Reply:
We had a discussion about that in the past (I don't remember if I have it somewhere, it might have been outside of Github), but we decided to hide that information in the UI. If you need to display superusers somehow and manage them through UI I'd recommend doing that in your fork. In Saleor I'd like to leave it as it was before and rely on permissions only.

Reply:
Sure, I've updated the PR.

I will expose superuser but disable editing it in my own fork. The reason for that is hiding the superuser information maks hard to find out why some staff can not edit other staff's profile who is superuser actually.